src/components/mfa/mfa-enroll.js

```diff
@@ -52,23 +54,43 @@ class MfaEnrollComponent {
 
   showComplete = () => {
     this.section = 'complete';
+    this.selectedFactor.isSubmitting = false;
     if (this.onComplete) {
       this.onComplete(this);
     }
   }
 
+
   showError = (err) => {
     if (err.error === 'invalid_request') {
       err.message = 'The code you entered was not valid.';
     }
 
+    if (err.message === 'An existing phone with that number is already associated with a factor for that Account.') {
```

Comment:
This is the message that comes back from Client API right? Almost smells like a good candidate for an error code. String matching feels icky. 😉